### PR TITLE
52: Standalone launch example

### PR DESCRIPTION
**Pull request by rserur**
- Oauth process very similar to existing EHR launch example.
- Built atop reorganize-launch-examples branch.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -23,7 +23,18 @@ send back a launch_context containing, among other things, an access token to
 set in the Authorization header and use for subsequent FHIR requests (to the
 ISS).
 
+## [examples/smart-standalone](./examples/smart-standalone)
+
+This provides the same routes above,
+but instead of an EHR launching from the `/launch` route, a user would directly
+visit the route with two different parameters: iss and scope. For example:
+
+`https://localhost:3000/launch?iss=http://example.com/fhir&scope=openid profile offline_access user/*.* patient/*.* launch/encounter launch/patient`
```
Comment (steve9001) [inline examples/README.md]: The spaces in the `scope` param look odd

File: examples/smart-standalone/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (steve9001) [inline examples/smart-standalone/package.json]: should we just remove the `"test"` key?

File: examples/README.md (hunk 1 above)
Comment (rserur) [reply examples/README.md]: `https://localhost:3000/launch?iss=http://example.com/fhir&scope=openid%20profile%20offline_access%20user%2F*.*%20patient%2F*.*%20launch%2Fencounter%20launch%2Fpatient`?

File: examples/smart-standalone/smart-standalone-launch.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+ *
+ * In this example, there are two routes:
+ *  - /launch
+ *  - /callback
+ *
+ * A user will visit the launch route independent of the EHR with two parameters,
+ # the ISS and scope. For example:
+ #
+ # https://localhost:3000/launch?iss=http://launch.smarthealthit.org/v/r2/sim/123/fhir
+ # &scope=openid profile offline_access user/*.* patient/*.* launch/encounter launch/patient
```
Comment (bkaney) [inline examples/smart-standalone/smart-standalone-launch.js]: I think here we should URL encode the spaces as well

**Pull request merged**